### DISCUSSION TITLE
post/professional-engineer-information-management-james/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#33] [Announcements]
# post/professional-engineer-information-management-james/

기술인으로 좀 더 재미있고 가치있는 삶으로 살기 위한 고민과 함께 성장하는 카카오페이 개발자들의 이야기

https://tech.kakaopay.com/post/professional-engineer-information-management-james/

**Comment by unme88** [comment]
대단합니다~!!!

**Comment by yozmngooddev** [comment]
기술사가 모든이에게 정답은 아니지만, 누구나 한번쯤은 고민했던 문제인것 같아요. 한번 더 생각하게 만든 글쓴이에게 감사드려요.

**Comment by ssyong73** [comment]
129회 동기회 회장님 ! 
화이팅 ~~ 진심으로 응원합니다. ^^
